src/utils/text.js

```diff
@@ -9,6 +9,9 @@ export const overviewTitleText = 'Overview'
 export const newLessonsTitleText = 'New'
 export const newLessonsActionText = 'Create a new lesson'
 
+export const newLessonsInstructionsDescriptionText = 'You can choose a requested topic below, or propose your own topic. Check out the guide portion on egghead lesson concepts for more details on proposing your own topics.'
```

Comment:
My issue with that is you loose the benefit of passing props for validating accepted types of data; and the automatic relative/absolute link handling and such.